src/client/component.js

```diff
@@ -3,8 +3,7 @@ const walkVars = require('./visitors/vars');
 const walkNode = require('./visitors/node');
 const utils = require('./utils');
 
-require('__IDYLL_SYNTAX_HIGHLIGHT__');
-let results = require('__IDYLL_AST__');
+// require('__IDYLL_SYNTAX_HIGHLIGHT__');
```

Comment:
I didn't investigate what this one does but maybe needs an alternate implementation?